scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -0,0 +1,304 @@
+        jar_file=jar_file,
+        mem=4,
+        tumor_hets=HetPulldown.tumor_hets,
+        called_file=TumorCaller.called_file,
+        tn_file=TumorNormalizeSomaticReadCounts.tn_file
+  }
+}
+
+# Pad the target file.  This was found to help sensitivity and specificity.  This step should only be altered
+#  by advanced users.  Note that by changing this, you need to have a PoN that also reflects the change.
+task PadTargets {
+    File target_bed
+    Int pd = 250
+    Int mem
+    File jar_file
+    command {
+        java -Xmx${mem}g -jar ${jar_file} PadTargets  --targets ${target_bed} --output targets.padded.bed --padding ${pd}  --help false --version false --verbosity INFO --QUIET false
+    }
+    output {
+        File padded_target_bed = "targets.padded.bed"
+    }
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+# Calculate the target coverage
+task CalculateTargetCoverage {
+    String entity_id
+    File padded_target_bed
+    String transform = "PCOV"
+    String grouping = "SAMPLE"
+    Boolean keepduplicatereads = true
+    Boolean disable_all_read_filters = false
+    File input_bam
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    Int mem
+    File jar_file
+    String disable_reference_validation
+
+    command {
+        java -Xmx${mem}g -jar ${jar_file} CalculateTargetCoverage --output ${entity_id}.coverage.tsv --groupBy ${grouping} --transform ${transform} --targets ${padded_target_bed} --targetInformationColumns FULL --keepduplicatereads ${keepduplicatereads} --input ${input_bam} --reference ${ref_fasta}  --disable_all_read_filters ${disable_all_read_filters} --interval_set_rule UNION --interval_padding 0 --secondsBetweenProgressUpdates 10.0 --disableSequenceDictionaryValidation ${disable_reference_validation} --createOutputBamIndex true --help false --version false --verbosity INFO --QUIET false
+    }
+    output {
+        File gatk_cnv_coverage_file = "${entity_id}.coverage.tsv"
+    }
+
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+task NormalizeSomaticReadCounts {
+    String entity_id
+    File coverage_file
+    File padded_target_bed
+    File pon
+    Int mem
+    File jar_file
+    File jni_lib
+    command {
+        java -Xmx${mem}g -Djava.library.path=${jni_lib} -jar ${jar_file} NormalizeSomaticReadCounts  --input ${coverage_file} \
+        --targets ${padded_target_bed} --panelOfNormals ${pon} --factorNormalizedOutput ${entity_id}.fnt.tsv --tangentNormalized ${entity_id}.tn.tsv \
+        --betaHatsOutput ${entity_id}.betaHats.tsv --preTangentNormalized  ${entity_id}.preTN.tsv  --help false --version false --verbosity INFO --QUIET false
+    }
+
+    output {
+        File tn_file = "${entity_id}.tn.tsv"
+        File pre_tn_file = "${entity_id}.preTN.tsv"
+        File betahats_file = "${entity_id}.betaHats.tsv"
+    }
+    #runtime {
+    #    docker: "gatk-protected/a1"
+    #}
+}
+
+task PerformSegmentation {
```

Comment:
Needs comment/doc.  It should note something about the unexposed (to the WDL script, that is) CBS parameters.